### PR TITLE
chore(sveltekit): update sveltekit eslint tooling

### PR DESCRIPTION
**Pull request by ndom91**
## ☕️ Reasoning

- Update svelte eslint tooling from `eslint-plugin-svelte3` to `eslint-plugin-svelte` ([Source](https://github.com/sveltejs/eslint-plugin-svelte))
- Added `dist` output folder to `clean` npm script in `framework-sveltekit`
- Cleaned up a few small things like rm ignoring directories that no longer exist and removing unnecessary prettier overrides

**I'd love someone with VSCode to test this as well though, looks good to me in nvim and build output is the same too**

## 🧢 Checklist

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged

## 📌 Resources

- [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md)
- [Contributing guidelines](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md)
- [Code of conduct](https://github.com/nextauthjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

File: packages/frameworks-sveltekit/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "build": "pnpm clean && pnpm providers && pnpm check && svelte-package",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
-    "clean": "rm -rf client.* index.* package src/lib/providers",
+    "clean": "rm -rf dist client.* index.* package src/lib/providers",
```
Comment (ThangHuuVu) [inline packages/frameworks-sveltekit/package.json:30]:
```suggestion
    "clean": "rm -rf dist src/lib/providers",
```
these are also no longer used

**Review by ThangHuuVu — COMMENTED** [review]
Should we also remove `next-auth/packages/frameworks-sveltekit/.eslintrc.cjs`, I think its not used right

**Comment by ndom91** [comment]
> Should we also remove `next-auth/packages/frameworks-sveltekit/.eslintrc.cjs`, I think its not used right

Yeah I think you're right. I've removed them :+1:

**Review by ThangHuuVu — APPROVED** [review]

**Comment by ThangHuuVu** [comment]
@ndom91 there is still one issue in the build, shall we fix it here too?

**Pull request merged**